pages/common/carp.md

```diff
@@ -0,0 +1,28 @@
+# carp
+
+> REPL and build tool for Carp.
+> More information: <https://carp-lang.github.io/carp-docs/Manual.html>.
+
+- Start a REPL:
```

Comment:
Based on the `node` page:
```suggestion
- Start a REPL (interactive shell):
```

Reply:
`tldr clj`

Reply:
I don't thinnk such minor details have to be standardized. Personally I don't think it's a bad idea to include  `(interactive shell)`, since not everyone knows what a `REPL` is.

Reply:
```
  git repl

  Git REPL (read-evaluate-print-loop) - an interactive Git shell.
  Part of `git-extras`.
```
  git repl

  Git REPL (read-evaluate-print-loop) - an interactive Git shell.
  Part of `git-extras`.

git repl is a good example, maybe we could build off of that

Reply:
I think that could be a good solution. In cases where we don't have much space, we could leave out the ` an interactive ...` part.